plugins/rest/rest.go

```diff
@@ -37,7 +40,7 @@ type Config struct {
 	Name                         string            `json:"name"`
 	URL                          string            `json:"url"`
 	Headers                      map[string]string `json:"headers"`
-	AllowInsureTLS               bool              `json:"allow_insecure_tls,omitempty"`
+	AllowInsecureTLS             bool              `json:"allow_insecure_tls,omitempty"`
```

Comment:
lol That's a breaking API change, isn't it? But let's do this one. If we wanted to be extra-eager, we could support both field names....

Reply:
Ahh, yeah. Is there somewhere I should note that this will be a breaking change?

Reply:
When squashing please make sure it's mentioned in the commit message. 🗒️

Reply:
Will do!